### PR TITLE
Add jade-run-chrome

### PR DESCRIPTION
**Pull request by TatriX**

### CODE REVIEW

File: jade-chrome.el [hunk 1]
```diff
@@ -35,10 +35,33 @@
 
 (require 'jade-webkit)
 
+(defvar jade-chrome-bin "chromium")
```
Comment (NicolasPetton) [inline jade-chrome.el]: This should be a `defcustom` instead, so the user can choose between Chrome and Chromium, and pick the binary of this system.

File: jade-chrome.el [hunk 2]
```diff
@@ -35,10 +35,33 @@
...
+(defvar jade-chrome-port "9222")
```
Comment (NicolasPetton) [inline jade-chrome.el]: Same here, it would be better as a `defcustom`.

Comment (NicolasPetton) [inline jade-chrome.el]: The best would be to have a "jade-chrome" custom group as a sub-group of the "jade" group.

File: jade-chrome.el [hunk 3]
```diff
@@ -35,10 +35,33 @@
...
+
+(defun jade-run-chrome (url)
+  "Start chromium with remote debugging enabled"
```
Comment (NicolasPetton) [inline jade-chrome.el]: You should end documentation sentences with a full stop.

If you're not already using it, I would recommand using flycheck, which is great for finding this typos (and many other linting issues & errors).

File: jade-chrome.el [hunk 4]
```diff
@@ -35,10 +35,33 @@
...
+  (interactive "sUrl: ")
+  (make-process :name "*jade-chromium*"
+                :command (list jade-chrome-bin
```
Comment (NicolasPetton) [inline jade-chrome.el]: as a safeguard (and for better issue reporting to the user) what about this:

``` elisp
(let ((executable (executable-find jade-chrome-executable)))
  (unless executable
    (user-error "Cannot find chrome/chromium binary in PATH."))
  (make-process ...))
```

File: jade-chrome.el (hunk 1 above)
Comment (NicolasPetton) [inline jade-chrome.el]: `jade-chrome-executable` would be a more idiomatic name.

File: jade-chrome.el [hunk 5]
```diff
@@ -35,10 +35,53 @@
...
+  :group 'jade)
+
+(defcustom jade-chrome-executable
+  "chromium"
+  "Chrome executable."
+  :type '(file))
+
+(defcustom jade-chrome-port
+  9222
+  "Chrome port."
```
Comment (NicolasPetton) [inline jade-chrome.el]: What about "Chrome remote debugger port"?

**Comment by NicolasPetton** [comment]
Looks good! Squash your commits and we're good to go!

**Comment by TatriX** [comment]
Should we mention new command in the readme?

**Comment by NicolasPetton** [comment]
TatriX <notifications@github.com> writes:

> Should we mention new command in the readme?

Yes!

**Pull request merged**